### ISSUE TITLE
Design Approach Document

### ISSUE THREAD

**Issue by awright** [hotosm/visualize-change#2]
This document will document project design approach and will (at minimum) include the following items:
- [ ] storyboard of workflow, 
- [ ] test for the initial wireframes

**Comment by awright** [comment]
Considering storyboard of workflow, @szymonkaliski put together a user workflow for our review:

# core flow

**1. user opens the website:**
  - user can navigate the map as usual
  - there's a toolbar which allows user to select 
    - [ ] date span
    - [ ] input lat/lon 
    - [ ] zoom level
    - [ ]  features visible on the map: buildings, roads, etc...(?)

**2. user selects part of the map and date span**
**3. changing the date updates the view in real time**
**4. user chooses to render the current view**
**5. modal window appears with options:**
  - lat/lon/zoom, date span and map features to confirm
  - animate daily/weekly/...(?)
  - size of the generated video (dropdown with few different options: 1280x720, 1024x768, ...)
  - input field for user email
  - "confirm"/"render" button

**6. after the offline rendering finishes, user gets notified by email with link to the MP4**
**7. MP4 is deleted after 7 days(?)**

# possible additional functionalities
- ability to select different map styles and features styles
- ability to add captions to the rendered MP4
- "gallery" of exported MP4s browsable by all users
- user accounts (login/password) - not sure if needed
- some kind of CMS with list of scheduled renders, history of exported files, etc...
- ?

**Comment by awright** [comment]
looks good. questions:
- @smit1678 can you confirm MP4 deleted after 7 days? (makes sense to me. could even be less)
- @szymonkaliski i thought captions were includes in the initial CFP. are they an additional feature or somewhere else?

**Comment by awright** [comment]
![screen shot 2018-01-25 at 15 07 59](https://user-images.githubusercontent.com/78410/35456254-d97a8e16-02a2-11e8-9ee7-a71a34d9b9ba.png) also relevant to #3

**Comment by szymonkaliski** [comment]
So my plan is to, together with @hoffmannjan, prepare the initial wireframes/mockups first (somewhere around first week of February), and after we discuss them, create polished design.

Regarding captions, I'm happy to add them after first alpha version (in March), I'd prefer to keep the "core flow" as "core" as possible.

And regarding the 7 days - this can as well be 30 or 360 or never, we will set this up as a config options somewhere so we can decide based on storage capacity of the server etc.

**Comment by awright** [comment]
Let's look at the contract schedule again regarding ui/design. I think what you have sketched out is a little later than we initially agreed but if the goal is a polished design by March, I think it will all work out. 

Here is a sketch of the UI/wireframe deadlines in the contract. 

- Jan 31: Design approach doc
- Feb 16: Alpha version of frontend
- Feb 28: Beta version of frontend
- March 31: Frontend user guide

Re: Captions: I agree with you here, let's hold off and focus on the core flow.

**Comment by szymonkaliski** [comment]
I’m pretty sure we can get fist wireframe design out in few days just so we can chat about it, hope that’s ok.

I think there’s nothing to worry about - we have the app basically working right now, and we have February to get it to look nice with the core flow, and March to get it polished :)

**Comment by awright** [comment]
Awesome 👍🎉

**Comment by szymonkaliski** [comment]
Here are some basic wireframes:

**1**

![1](https://user-images.githubusercontent.com/1040420/35624532-8ee45828-068f-11e8-9705-aa5d7be6be42.png)

**2**

![2](https://user-images.githubusercontent.com/1040420/35624533-8f708154-068f-11e8-8a7e-428dae21df64.png)

**3**

![3](https://user-images.githubusercontent.com/1040420/35624534-8fa1d484-068f-11e8-9a5a-8f24c0789396.png)

1&3 are the core flow, 2 shows inspector view for applying custom styles to map; let's focus on 1&3 for now, with addition of toggling roads/buildings/etc(?) for rendered video, and tackle custom styles after that

**Comment by szymonkaliski** [comment]
Just a reminder, we would like some comments / "green light" on the wireframes here, then we will create designs based on those wireframes, and then actual working version of the frontend.

**Comment by smit1678** [comment]
Couple questions about the flow and the way a user will interact, and maybe we need a bit more of a narrative to match the wireframes to explain some things so I understand better: 

I think part of the original vision was that the view in the browser would serve as the primary way to visualize the changes but that you can also render it out into movie for offline usage. Part of what I get from the flow and wireframes at the moment is that the user progresses towards rendering as the goal. I think that's a secondary feature (even though it's the toughest to implement). What are the ways that we could emphasize the browser as the primary viewer (related to #1 and our technical approach and decision). 

Do we need to have all of the same options in the render modal? Is there a reason for this? Could the options like map, start/stop, date selector all be on the primary view? Then the modal is simplified just to the render related functions. 

What does the map settings section do? Will a user interact with them for specific settings or will it just exposing this information to the user? What was the vision there? 

Are there ways we could improve the hierarchy a bit more? As a first time user, I will know that I can interact with the map, but I might need some more information in the sidebar to orient me and tell me what to do. 

As a return user, is it a better experience to have to recreate your visualization each time or have a specific URL they can go back to or share (prob also related to #1)? 

Instead of Render, do we want to look to have the action be Share or Save? Then the modal can be share or save options for what a user does with the visualization they created. I do think rendering as a movie file is an option but not our only option. 

I like the potential for custom styles but I also agree that is non-core and can be addressed later.

**Comment by smit1678** [comment]
> And regarding the 7 days - this can as well be 30 or 360 or never, we will set this up as a config options somewhere so we can decide based on storage capacity of the server etc.

Related to some of the questions in my comment above, I guess is there a reason we're keeping it around at all? Do we want to consider dropping it right away after download? Or should it be more of a key thing that is saved with the view so others can download?

**Comment by mataharimhairi** [comment]
I think we should try keeping it for 7 days initially so that it gives the users a chance to return and download in case they delete/lose it by accident, and/or want to share the download link. This might end up reducing the processing load on the tool, as users won't be re-creating the same visualisation since it was instantly dropped.

**Comment by mataharimhairi** [comment]
A couple initial thoughts/comments on the overall core workflow:

1. Agree with Nate regarding the ability to visualise the changes in the browser, before rendering into a movie for offline use. This will help reduce the unnecessary creation of videos, if the user can see the map changes in the browser first to decide whether or not it is suitable for their needs.
2. I think we should include the requirement to login and authenticate with OSM usernames for the tool, as it will help to monitor and control the use of it to some extent.
3. There should be additional options to select the area of interest for rendering. Entering coordinates is one of the least user friendly ways to do this. Ideally the options available in the Export Tool would be implemented here as well:
a.) search by location or coordinates
b.) bounding box or free draw
c.) select the current view
d.) import a polygon

<img width="1392" alt="screen shot 2018-02-02 at 13 41 19" src="https://user-images.githubusercontent.com/8972913/35720243-612898b8-081f-11e8-8062-7759d12c2899.png">

An additional option that we spoke about, is the ability to use the area of interest from a specific Tasking Manager project, similar to OSM Analytics. This would be an extremely useful feature for those setting up mapping projects, to share the progress/results of the collective efforts.

**Comment by szymonkaliski** [comment]
Thanks for the comments!

> Do we need to have all of the same options in the render modal? Is there a reason for this?

The only reason in our thinking was to have user "confirm" the render, but we can drop this, as they are exactly the same as options in the sidebar, and just keep the rendering options in there.

> What does the map settings section do? Will a user interact with them for specific settings or will it just exposing this information to the user? What was the vision there?

This section shows the current map position/zoom, but fields are also editable and changing them reflects on the map, so our thinking was that some users will know exactly the lat/lng they want to render, and some would like to first position the map, and maybe tweak the values after that. Also, combining this with what @mataharimhairi said, we could add there search option.

> Are there ways we could improve the hierarchy a bit more? As a first time user, I will know that I can interact with the map, but I might need some more information in the sidebar to orient me and tell me what to do.

Sure, good point.

> As a return user, is it a better experience to have to recreate your visualization each time or have a specific URL they can go back to or share (prob also related to #1)?

We could store the values into URL, or DB, and generate links for returning users, what do you think?
This touches on @mataharimhairi point on logging in and authenticating with OSM usernames, then we probably should have a list of previously saved "views" etc...? Not sure if we can squeeze all of that in, into the February deadline for alpha version though.

> Instead of Render, do we want to look to have the action be Share or Save? Then the modal can be share or save options for what a user does with the visualization they created. I do think rendering as a movie file is an option but not our only option.

Sure, what other options are you thinking about?
We could relatively easily:
- render MP4 (already done)
- render GIF
- share as an iframe embed (although I'd prefer not to overpromise that this would be ready by the end of February)
- share link to current "view"
Anything other comes to mind?

> Agree with Nate regarding the ability to visualise the changes in the browser, before rendering into a movie for offline use. This will help reduce the unnecessary creation of videos, if the user can see the map changes in the browser first to decide whether or not it is suitable for their needs.

So the idea is, while user plays with the sidebar, the changes are updated live in the map view, so it is visualised live on the frontend always, and rendering to mp4 is just an option (but I agree, it's emphasized currently).

> I think we should include the requirement to login and authenticate with OSM usernames for the tool, as it will help to monitor and control the use of it to some extent.

I already touched on that in previous answer, we could add login/auth, but I'd prefer not to focus on that in the "core flow" that we aim to have at the end of February - maybe it's worth making another issue about that, and get back to it in March?

> There should be additional options to select the area of interest for rendering. Entering coordinates is one of the least user friendly ways to do this. Ideally the options available in the Export Tool would be implemented here as well:
a.) search by location or coordinates
b.) bounding box or free draw
c.) select the current view
d.) import a polygon
> An additional option that we spoke about, is the ability to use the area of interest from a specific Tasking Manager project, similar to OSM Analytics. This would be an extremely useful feature for those setting up mapping projects, to share the progress/results of the collective efforts.

Ok, this complicates things a bit, so, currently the filtering is done in "online-processing" fashion, where the timestamp (date) and visible features are filtered using mapboxgl filter options. I'm honestly not sure right now, how complex it would be to filter by bounding box, or complex shape/polygon.
I'd propose, for "core flow", to keep the render centered around current lat/lng, with option to select few different resolutions, and we could come back to more complex options later.

So, to recap:

- we will update the wireframes to reflect some of the comments:
  - simplified user flow
  - some kind of help/live tutor to guide the user a bit more
  - making "live map updates" the core functionality, and render the secondary
  - adding "search" field for map
- we should create two new issues:
  - one tracking the possible user logging
  - one tracking the possible complex export options (bounding box, polygon, import from task manager)
- we should discuss (these might not make into the "core flow" for February deadline and be added later!):
  - how to handle rendered files? (delete after download, keep for some time, keep forever)
  - what "save/export" options we want ("mp4", "gif", "iframe", "url", ...?)
  - are we adding login/auth (this could potentially mean that user has access to list of previously created maps"

**Comment by smit1678** [comment]
fwiw because I didn't say this in my earlier comment, I think the progress we're making is great and super happy we're digging into these questions and decisions. 

> so, currently the filtering is done in "online-processing" fashion, where the timestamp (date) and visible features are filtered using mapboxgl filter options. I'm honestly not sure right now, how complex it would be to filter by bounding box, or complex shape/polygon.

I think there are two things being discussed here: 

1. Searching, centering, navigating the map
2. Clipping/filtering the final visualization view

For searching and centering the map, I don't think entering lat/lon is going to be a good experience. I've never used that and have repeatedly encountered the question - how do I enter a city or a place to quickly navigate - when doing user testing on other tools. I think part of @mataharimhairi's point was that having other options for navigating and centering the map is helpful for a user. So I think a search field will be helpful - can we just combine the lat/lon into this search bar and simplify everything?

Agreed that second one is more complex and may not be worth time investment up front right now. While super helpful to clip a visualization by a Tasking Manager AOI, it might not be the core focus for this first phase. 

Here is an example of [Task #3322](https://tasks.hotosm.org/project/3322) that is used as a way to show the area within the osm-analytics application: http://osm-analytics.org/#/show/hot:3322/buildings/recency. 

> We could store the values into URL, or DB, and generate links for returning users, what do you think? This touches on @mataharimhairi point on logging in and authenticating with OSM usernames, then we probably should have a list of previously saved "views" etc...? Not sure if we can squeeze all of that in, into the February deadline for alpha version though.

I think that's fine but goes back to the technical approach and making sure we're architecting this up front to handle major features later in March or something new in a next phase. We're all in agreement that polishing the core flow should be the focus for February but we want to make sure that the design and architecture decisions that are being made now are based on understanding what is possible or not possible / what should be done versus should not be done for a good experience. 

Let's decide what's the best approach both for user experience but also for maintenance and hosting considerations. 

We'll break some of those other discussion items into separate tickets.

**Comment by hoffmannjan** [comment]
There is some design update based on your comments.

**1**
![1](https://user-images.githubusercontent.com/3756160/35739144-f6f3c636-0830-11e8-8186-22d25a7e1670.png)
**2**
![2](https://user-images.githubusercontent.com/3756160/35739149-fa62a076-0830-11e8-89e5-cba6fd8d85ed.png)
**3**
![3](https://user-images.githubusercontent.com/3756160/35739152-fc8f94b2-0830-11e8-80ae-8c0563afedfe.png)
**4**
![4](https://user-images.githubusercontent.com/3756160/35739156-fe8296c0-0830-11e8-842d-7e097f4f3bd2.png)
**5**
![5](https://user-images.githubusercontent.com/3756160/35739158-002f2600-0831-11e8-8a82-77661b6aec23.png)


Looking forward for more directions :).

**Comment by awright** [comment]
@hoffmannjan also glad we are talking here and I think there has been very good progress. I had one (maybe) small question for everyone cc @smit1678 @mataharimhairi 

I don't think the main tabs of "Base Settings" and "Visual Settings" are helpful. Is that generally agreed to? I'm also not clear as to why we need the two?

I find the lat/long search unnecessary. I think the first section on "Base Settings" should be "Search"  and search by bounding box or geography options. The second section leading you through style and frame rate. I think the style options should be higher priority (aka second after search) than selection of frame rate, and then a preview"

Why not have those as the main tabs on top. So something like (and just a suggestion here)
- Search
- Style
- Dates
- Preview

It would be good to see further evolutions and options for design. And we can further discuss next week on the phone. Will add people to invite as optional attendees. cc @szymonkaliski @mataharimhairi @smit1678 @hoffmannjan

**Comment by hoffmannjan** [comment]
@awright this is version with changes, take a look
![new](https://user-images.githubusercontent.com/3756160/35744651-2c696d42-0841-11e8-8ee2-6caf95bd7be3.png)

**Comment by awright** [comment]
@hoffmannjan I think it's looking good! Another further question -- how do you see the Style setting being a preview? 

I hope this isn't too specific for what we are talking about here -- but I rather like something like the "Start mapping now" button https://www.mapbox.com/bites/00343/#5.8/-19.036/24.323

Not the same language but it's pretty visible. So a specific "preview" button. And then the share, export, play pause more like the regular share buttons as found on Giphy and YouTube. https://giphy.com/gifs/JIX9t2j0ZTN9S https://www.youtube.com/watch?v=5dsGWM5XGdg

How about put them below the map as an overlay?

**Comment by szymonkaliski** [comment]
What would the "preview" button do? Any change to the options in sidebar is reflected in the map on the right (hopefully almost) instantly, so I don't think we need another button here.

I also would prefer to keep as much as possible in the sidebar, just because we can later add a button to hide it behind left part of the screen, keeping only the map visible :)

**Comment by awright** [comment]
I think you can think of the share and preview options like the bottom area of a youtube video. it might be helpful to think about the priorities of the options. the search, date range, style sections are like the settings of the animation. the sharing, download, preview options are important in functionality, but also secondary in terms of interaction. i really think youtube could be helpful here but this will be much easier to talk about in person, so let's make sure to discuss this flow tomorrow. please add to the #21 meeting agenda too!

**Comment by hoffmannjan** [comment]
![wire1](https://user-images.githubusercontent.com/3756160/35922999-5f2bf394-0c1f-11e8-81c7-49b9938398b3.png)
![wire2](https://user-images.githubusercontent.com/3756160/35923004-613d3738-0c1f-11e8-9f67-d4bb1d9fac5f.png)

our new proposition for the export / share flow.

**Comment by awright** [comment]
<img width="1440" alt="screen shot 2018-02-07 at 10 30 22 am" src="https://user-images.githubusercontent.com/78410/35924788-0995402a-0bf2-11e8-94ce-1bdf9abda690.png">
<img width="1440" alt="screen shot 2018-02-07 at 10 30 28 am" src="https://user-images.githubusercontent.com/78410/35924789-09a6fe28-0bf2-11e8-99d6-ae6bcc23a9d5.png">

**Comment by awright** [comment]
Plan for next week:
- Updates designs with #22 meeting feedback 
- Start to code this interaction in demo http://165.227.230.157:8080/

**Comment by hoffmannjan** [comment]
Hi, 

our proposition for application ui design
![desktop hd](https://user-images.githubusercontent.com/3756160/36102087-70ba771c-100b-11e8-9438-d18461cd2dda.png)

Looking forward for your approval so we can start implementing this.

**Comment by awright** [comment]
Looking really great @hoffmannjan !! It's really important for people to know where they are in the process of making and viewing these animation - so things like titling, date, feature -- are very important. I see you are doing that work already and I'd like to pull out these pieces even ore. 

Also, it's probably useful to remember that there are two views for this animation 1) the viewer and 2) the creator. Maybe we can start to organize our design wireframes in these two categories? The two views should be very similar, but I want to make sure we are designing for both use case. 

You can see some of my conceptual suggestions in the PDF below. Though we may not go in this direction, I'd like to see one option -- putting the settings options in tabs and not as a vertical side bar. 

Also - if you are putting any design aesthetic on the work, I would encourage us to be a little more modern. Again - refer to the Mapbox Animation, ITO, and even Giphy creation. I think successful animation workflows are bold and would encourage us to do that. 

![animation design feedback 001](https://user-images.githubusercontent.com/78410/36108365-b3f11ff2-0fea-11e8-80b2-28a9f6142681.jpeg)

[Animation Design Feedback.pdf](https://github.com/hotosm/visualize-change/files/1717051/Animation.Design.Feedback.pdf)

**Comment by szymonkaliski** [comment]
@awright thanks for the feedback

0. this is design for tool in "creating the animation" mode, as we discussed, when visiting this as a "look-only" user, the sidebar disappears
1. we already agreed on vertical sidebar, so I'm not sure what to do about that tab note; I think we should stick with our decisions, since it was discussed few times already?
2. date span is collapsible vertically already (notice that triangle on the right)
3. we can change the word "export" to "share", not a problem
4. I'm open to better name than "interval" if you have any suggestions
5. we will make date/time more prominent in the lower right corner
6. legend is a good idea, but again, my opinon is that text is not something that we should add for video/gif exports, because it scales badly (especially if it's small)
7. we already discussed captions as something to be outside of "core flow" and to be discussed after the beta release at the end of February
8. from what I checked we can't count currently visible features for the upper-right "buildings mapped by..." - for malaria mapping project this was done with pre-processing step, that we can't do in this tool
9. not sure if I understand note about the header, do you mean the red background is a problem, or that OpenStreetMap Analytics has different font or combination of those, or something else? as far as I understood OSM doesn't have a CI document, right?

If there's no other notes, I think the changes are small enough that we could start working on the implementation, or would you like to see updated design?

**Comment by awright** [comment]
Notes inline. Thanks for the quick reply! 

1. this is design for tool in "creating the animation" mode, as we discussed, when visiting this as a "look-only" user, the sidebar disappears

Please indicate that in wireframes. 

2. we already agreed on vertical sidebar, so I'm not sure what to do about that tab note; I think we should stick with our decisions, since it was discussed few times already?

I'd like to see a vertical sidebar with tab options. Please update. 

3. date span is collapsible vertically already (notice that triangle on the right)

Great. That entire settings side bar needs to be collapsible too. 

4. we can change the word "export" to "share", not a problem

👍 ;) 
5. I'm open to better name than "interval" if you have any suggestions

Duration? Frame Rate? (Would recommend doing some research. What does Filemaker Pro call it?)

6. we will make date/time more prominent in the lower right corner

I'd also like to see a version that has something like the Mapbox date and feature highlight. 

7. legend is a good idea, but again, my opinon is that text is not something that we should add for video/gif exports, because it scales badly (especially if it's small)

Then please add something like the Mapbox date and feature highlight as described above. Let's also discuss on Wed. 

8. we already discussed captions as something to be outside of "core flow" and to be discussed after the beta release at the end of February

I'd like to review this

9. from what I checked we can't count currently visible features for the upper-right "buildings mapped by..." - for malaria mapping project this was done with pre-processing step, that we can't do in this tool

Don;'t take that literally -- conceptually it's an indication of what users are watching in the animation. If I am a random person coming to this animation - what am I seeing?

10. not sure if I understand note about the header, do you mean the red background is a problem, or that OpenStreetMap Analytics has different font or combination of those, or something else? as far as I understood OSM doesn't have a CI document, right?

Let's discuss this on Slack. In general, "HOT Visualization Change" is not the best title for the page. So let's layout something different.

**Comment by szymonkaliski** [comment]
Here's options with updates from your comments:

![1](https://user-images.githubusercontent.com/1040420/36195561-29c37ff2-116e-11e8-97de-3bc504dc1922.png)

![0](https://user-images.githubusercontent.com/1040420/36195560-299ffece-116e-11e8-949e-019b11102cca.png)

>> 2. we already agreed on vertical sidebar, so I'm not sure what to do about that tab note; I think we should stick with our decisions, since it was discussed few times already?

>I'd like to see a vertical sidebar with tab options. Please update.

Please notice that one of our first wireframes had tabs: https://github.com/hotosm/visualize-change/issues/2#issuecomment-362607470 and based on your comments (https://github.com/hotosm/visualize-change/issues/2#issuecomment-362685466 "I don't think the main tabs ... are helpful.") we changed it to all vertical collapsible one: https://github.com/hotosm/visualize-change/issues/2#issuecomment-362639666
If we keep coming back to what we agreed on it will be impossible to make progress. 

Also, if we ignore all of that, imagine if we have tab for just the "date span" how would it look like? "date span" at the top, and then rest of the sidebar empty, which feels like huge waste of space...?

Also, this design is for the first "beta" iteration at the end of February, we should gather user feedback then and we can change the UI to tabs or floating panes, or modals, or something else based on that data then, but for now we have to settle on something and implement to meet the deadline.

> > 5. I'm open to better name than "interval" if you have any suggestions

> Duration? Frame Rate? (Would recommend doing some research. What does Filemaker Pro call it?)

It's not duration (length of the animation), and not frame rate (how many frames per second), so these do not fit here, or will be confusing. I never used "filemaker pro" and not sure what that is really... Only thing that I found with this name is some expensive relational database, but I'm pretty sure you didn't mean that, right?
I think "interval" fits here, because as per definition it's "a space of time between events or states" which is exactly what it is here - how big of a step the tool takes, do we jump in hours, days, or weeks... Again, I'm happy to use different word if you have any suggestions.

>> 10. not sure if I understand note about the header, do you mean the red background is a problem, or that OpenStreetMap Analytics has different font or combination of those, or something else? as far as I understood OSM doesn't have a CI document, right?

> Let's discuss this on Slack. In general, "HOT Visualization Change" is not the best title for the page. So let's layout something different.

We've left the header empty for now, while waiting for proper title.

**Comment by awright** [comment]
Things are looking great and there is great work being done, but I believe
there is a misunderstanding here about HOT design feedback. Let’s dig into
this during the meeting and get to an agreed upon list of design changes we
would like to see in the next week.

I will not have any time before our meeting to review ticket updates, so if
there are any further topics or items to look at, let’s make sure to
allocate time and address them during the meeting. Thanks!

**Comment by awright** [comment]
Let's see some designs that incorporate the following:
- Date written out eg May 15, 2019 09:00AM
- The left sidebar organized into three sections: 1) search 2) date 3) style
  - The Style Section would include the current Map, Buildings and Roads sections

When a user comes to the page to make an animation, they with all main functionality sections collapsed.

**Comment by awright** [comment]
@szymonkaliski @hoffmannjan to provide additional UI designs for the tool with same user functionality we have discussed to date. suggesting a graphic design aesthetic more in line with current animations include mapbox animation, ITO and giphy.

**Comment by szymonkaliski** [comment]
Here's updated proposition based on comments from https://github.com/hotosm/visualize-change/issues/2#issuecomment-365644920

all sections opened:

![x1](https://user-images.githubusercontent.com/1040420/36247830-badfa872-1235-11e8-9cce-ab78c7ef08df.png)

all sections collapsed:

![x2](https://user-images.githubusercontent.com/1040420/36247834-bd56635c-1235-11e8-9c66-f7a8b8e5cabb.png)

We hope to have new colorscheme based on our chat around mon/tue.

**Comment by szymonkaliski** [comment]
Here's promised design proposition with dark colors:

![desktop hd](https://user-images.githubusercontent.com/1040420/36372470-7b097a92-1565-11e8-82a8-820e8f6ce48a.png)

**Comment by awright** [comment]
@szymonkaliski in our meeting last week, we discussed having a different aesthetic to the same user experience. has anything changed here beyond the color palette? perhaps tangram play can be more of an inspiration here. https://mapzen.com/tangram/play/#3.00/0.00/0.00

**Comment by szymonkaliski** [comment]
@awright from what I remember you've said that functionality should not change, and that you'd like to see the design in different, darker colours..? Not sure how tangram relates to this really?

**Comment by awright** [comment]
i asked for another design option. you could keep the same user experience but the look and feel of the wireframes (as seen above) needed to be different. let's talk later today if this is not clear.

**Comment by szymonkaliski** [comment]
@awright sorry, I can't chat today, we could schedule a meeting for tomorrow if you'd like.

I want to again touch on the issue that we all (I believe) settled on sidebar with sections and options that are currently visible, so to stay within that assumptions, we can only change colors/fonts/etc. do you mean something different by "look and feel" then? 
If we want to move stuff around and design different user experience, then I'd like to get some confirmation from @smit1678 and @mataharimhairi that it makes sense to take bigger step back here (and get bit more detailed feedback as to what you'd all like to see / how you imagine this differently as well). We'd have to move back to wireframes, and figure out what other layout of elements, panes, etc. makes sense for the tool first.
If I'm misunderstanding what you are saying, then could you maybe rephrase or write up a bit more what you have in mind?

**Comment by szymonkaliski** [comment]
Should we close this issue?
